### PR TITLE
Add moonbeam networks informations

### PR DESCRIPTION
**Pull request by librelois**
This PR add information for Moonbeam, Moonriver and Moonbase networks.

### CODE REVIEW

File: chain_info/Moonbase.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Moonbase Alpha
+network_type: testnet
+para_id: 1000
+chain:
+  type: parachain
+  parent: Moonbase Relay (based on Westend)
+  consensus: POS
+  sudo: true
```
Comment (chevdor) [inline chain_info/Moonbase.yaml:8]: The format has changed and para_id slides down inside `chain`.

```suggestion
chain:
  type: parachain
  parent: Moonbase Relay (based on Westend)
  consensus: POS
  sudo: true
  para_id: 1000
```

File: chain_info/Moonbeam.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+name: Moonbeam
+network_type: mainnet
+para_id: 2004
+chain:
+  type: parachain
+  parent: Polkadot
+  consensus: POS
+  sudo: false
```
Comment (chevdor) [inline chain_info/Moonbeam.yaml:8]:
```suggestion
chain:
  type: parachain
  parent: Polkadot
  consensus: POS
  sudo: false
  para_id: 2004
```

File: chain_info/Moonbase.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+name: Moonbase Alpha
+network_type: testnet
+para_id: 1000
+chain:
+  type: parachain
+  parent: Moonbase Relay (based on Westend)
```
Comment (chevdor) [inline chain_info/Moonbase.yaml:6]: This **must** match a valid name relay chain name.

File: chain_info/Moonbase.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
+name: Moonbase Alpha
```
Comment (chevdor) [inline chain_info/Moonbase.yaml:1]: This must match the file name. So you need to edit the filename or this value.

File: chain_info/Moonriver.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
+name: Moonriver
+network_type: mainnet
+para_id: 2023
+chain:
+  type: parachain
+  parent: Kusama
+  consensus: POS
+  sudo: false
```
Comment (chevdor) [inline chain_info/Moonriver.yaml:8]:
```suggestion
chain:
  type: parachain
  parent: Kusama
  consensus: POS
  sudo: false
  para_id: 2023
```

File: chain_info/Moonbase.yaml (hunk 3 above)
Comment (chevdor) [reply chain_info/Moonbase.yaml:6]: It sounds like you miss one file for `Moonbase` (the relaychain).

**Review by chevdor — CHANGES_REQUESTED** [review]
Looking good but a few changes are required.
Noticably: The `name` field **must** match the filename and `parent` must match a valid `name`.

File: chain_info/Moonbase.yaml (hunk 4 above)
Comment (librelois) [reply chain_info/Moonbase.yaml]: file renamed

File: chain_info/Moonbase.yaml (hunk 3 above)
Comment (librelois) [reply chain_info/Moonbase.yaml]: I added a file for Moonbase Relay

File: chain_info/Moonbase_Relay.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    type: relaychain
+    # parent: n/a
+    consensus: Aura
+    sudo: true
+chainspec:
+    http_url: https://github.com/PureStake/moonbeam-networks/blob/master/networks/alphanet/westend-raw-specs.json
+# parent: n/a
+goals:
+    - Relay chain for Moonbase Alpha and other parachains who want to tests in interactions with moonbase.
+repository: https://github.com/paritytech/polkadot
```
Comment (chevdor) [inline chain_info/Moonbase_Relay.yaml:13]: That one is probably not correct

File: chain_info/Moonriver.yaml [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    url: wss://wss.api.moonriver.moonbeam.network
+  - name: OnFinality
+    url: wss://moonriver.api.onfinality.io/public-ws
+  - name: PinkNode
+    url: wss://public-rpc.pinknode.io/moonriver
+api_endpoints:
+    - name: subscan
+      url: https://moonriver.subscan.io/
+    - name: blockscout
+      url: https://blockscout.moonriver.moonbeam.network/
```
Comment (chevdor) [inline chain_info/Moonriver.yaml:47]: Missing url

File: chain_info/Moonriver.yaml [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+    url: wss://moonriver.public.blastapi.io
+  - name: Moonbeam Foundation
+    url: wss://wss.api.moonriver.moonbeam.network
+  - name: OnFinality
+    url: wss://moonriver.api.onfinality.io/public-ws
+  - name: PinkNode
+    url: wss://public-rpc.pinknode.io/moonriver
+api_endpoints:
+    - name: subscan
+      url: https://moonriver.subscan.io/
```
Comment (chevdor) [inline chain_info/Moonriver.yaml:45]: According to https://support.subscan.io/#api-endpoints
that would be more something like `moonriver.api.subscan.io`

**Review by chevdor — CHANGES_REQUESTED** [review]

File: chain_info/Moonbase_Relay.yaml (hunk 6 above)
Comment (librelois) [reply chain_info/Moonbase_Relay.yaml]: We use the polkadot repo, sometimes our fork, but in practice we do not change anything.

File: chain_info/Moonriver.yaml (hunk 7 above)
Comment (librelois) [reply chain_info/Moonriver.yaml:47]: I don't understand this comment

**Comment by librelois** [comment]
@chevdor It seems to me that I have addressed all your remarks, can you do a final review and merge if everything is good?

**Review by chevdor — APPROVED** [review]

**Pull request merged**